keyboards/yncognito/batpad/readme.md

```diff
@@ -0,0 +1,15 @@
+# batpad
+
+![batpad](https://imgur.com/CZ3m0jh)
+
+A short description of the keyboard/project
```

Comment:
Please fill this out.

Reply:
![image](https://user-images.githubusercontent.com/40719920/84329789-daff1c80-abaf-11ea-998f-8c426dfafcb3.png)
I have already changed it